<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	background-color:rgb(227, 227, 227);
	border:1px solid;
	border-radius:8px;
	font: 14pt &quot;Yagut&quot;;
	padding:5px;
}


QTextEdit{
	background-color:rgb(227, 227, 227);
	border:1px solid;
	border-radius:8px;
	font: 14pt &quot;Yagut&quot;;
	padding:5px;
}

QLabel{
	font: 14pt &quot;Yagut&quot;;
	color:rgb(227, 227, 227);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/postman.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>821</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 52), stop:1 rgba(0, 0, 0, 56))</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="subject">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="reciever">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reciever</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/Gmail-logo-500x281.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
color:#000;</string>
   </property>
   <property name="text">
    <string>Ok now pass your mail information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:25px;
color:#000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/bubble-160786.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="message">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yagut'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Yagut&quot;;
	color:rgb(227, 227, 227);
	border:1px solid rgb(227, 227, 227);
	border-radius:8px;
}

QPushButton:hover{
background-color:rgb(51, 167, 82);
border:1px solid rgb(227, 227, 227);
}</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>29</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Yagut&quot;;
	color:rgb(227, 227, 227);
	border:1px solid rgb(227, 227, 227);
	border-radius:8px;
}

QPushButton:hover{
background-color:rgb(233, 66, 52);
border:1px solid rgb(227, 227, 227);
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="insert_file_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Yagut&quot;;
	color:rgb(227, 227, 227);
	border:1px solid rgb(227, 227, 227);
	border-radius:20px;
}

QPushButton:hover{
background-color:rgb(68, 140, 255);
border:1px solid rgb(227, 227, 227);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/icons8-attach-100.png</normaloff>../images/icons8-attach-100.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>47</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="files_tree">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Yagut&quot;;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>click to remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="loading">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>subject</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>reciever</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>error</zorder>
  <zorder>message</zorder>
  <zorder>label_5</zorder>
  <zorder>send_btn</zorder>
  <zorder>back_btn</zorder>
  <zorder>insert_file_btn</zorder>
  <zorder>files_tree</zorder>
  <zorder>label_8</zorder>
  <zorder>loading</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
